sktime/regression/dummy/_dummy.py

```diff
@@ -45,6 +45,17 @@ class DummyRegressor(BaseRegressor):
         The quantile to predict using the "quantile" strategy. A quantile of
         0.5 corresponds to the median, while 0.0 to the minimum and 1.0 to the
         maximum.
+
+    Examples
+    --------
+    >>> from sktime.regression.dummy import DummyRegressor
+    >>> from sktime.datasets import load_unit_test
+    >>> X_train, y_train = load_unit_test(split="train")
+    >>> X_test, y_test = load_unit_test(split="test")
+    >>> regressor = DummyRegressor(strategy="median") # doctest: +SKIP
```

Comment:
we don't strictly need to skip the doctest here, since `DummyRegressor` does not have an additional package dependency - but it is also not wrong since fit/predict is also tested by the test framework already so this would be a duplication.

Just FYI, nothing that needs changing.